komga/src/main/kotlin/org/gotson/komga/application/tasks/TaskHandler.kt

```diff
@@ -67,6 +69,11 @@ class TaskHandler(
             seriesRepository.findByIdOrNull(task.seriesId)?.let {
               metadataLifecycle.aggregateMetadata(it)
             } ?: logger.warn { "Cannot execute task $task: Series does not exist" }
+
+          is Task.EmptyTrash -> {
+            seriesRepository.findAllDeleted().let { seriesLifecycle.deleteMany(it) }
+            bookRepository.findAllDeleted().let { bookLifecycle.deleteMany(it) }
```

Comment:
Those operations should be bundled in a Lifecycle service instead of accessing raw repositories.